.github/workflows/testcafe_tests.yml

```diff
@@ -62,9 +62,9 @@ jobs:
       fail-fast: false
       matrix:
         ARGS: [
-          { componentFolder: "accessibility", name: "accessibility" },
-          { componentFolder: "accessibility", name: "accessibility - material", theme: 'material.blue.light' },
-          { componentFolder: "accessibility", name: "accessibility - fluent", theme: 'fluent.blue.light' },
+          { componentFolder: "accessibility", name: "accessibility", concurrency: 2 },
```

Comment:
Why?

Reply:
We tried to check if the test execution time would decrease if we set the value to 2. We can remove this value now, as there is no noticeable effect.

Reply:
[f3e0583b6c68591e7ea000377b1d886c0e1f85ee]